README.md

```diff
@@ -8,7 +8,7 @@
 Hey yo! [learnyouahaskell.github.io](https://learnyouahaskell.github.io/) is an **open-source** fork of the original **Learn You a Haskell** (**LYAH** for short) guide for Haskell by Miran Lipovača, "the funkiest way to learn Haskell, the best functional programming language around". This guide is meant for people who have programmed already, but have yet to try functional programming.
 
 **Now, why did I create this fork?** ("fork" means a copy of an original which may be modified or extended)
-Well, I began learning Haskell in 2021 at my undergrad studies and quickly came across LYAH ([link for original](http://learnyouahaskell.com/)). I immediately fell in love and followed it to supplement my journey of learning Haskell.
```

Comment:
This link is **legally required** for this project. Can be swapped with a https://web.archive.org/ version, but it cannot be removed.